projects/plugins/jetpack/modules/search/instant-search/components/path-breadcrumbs.scss

```diff
@@ -10,6 +10,10 @@
 
 .jetpack-instant-search__path-breadcrumb-link {
 	text-decoration: none;
+	-webkit-line-clamp: 1;
```

Comment:
>It only works in combination with the display property set to -webkit-box or -webkit-inline-box and the -webkit-box-orient property set to vertical. https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp

Just wondering whether this property works if not combined with the above properties. Because when I removed the fallback, it stops working in Safari which is webkit based, isn't it?

Reply:
Thanks for the feedback. After looking a little more into the docs it does seem that it would prefer to have the  `-webkit-box` or `-webkit-inline-box` and the `-webkit-box-orient` properties set, and that line-clamp does have a bit of a history of being less browser stable than some other properties. 

It looks like replacing line-clamp with max-width 100% has the same effect, but in a more browser-stable way without requiring the other properties. I've just pushed this change through now.

Reply:
Using `max-width: 100%` seems reasonable here, given that `.jetpack-instant-search__path-breadcrumb` basically uses the same styles.